.circleci/config.yml

```diff
@@ -85,6 +85,8 @@ jobs:
           command: |-
             # Because we are running this stage in a non-standard container we
             # need to specify the full path
+            python3 -m ensurepip
```

Comment:
We sshed into the circle ci env and ran the release manually. These were the steps needed to fix the pipeline.